cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -26,12 +33,20 @@ def _export(dst_file, temp_dir, instance_data, save_images=False):
 
     make_zip_archive(temp_dir, dst_file)
 
-@importer(name='YOLO', ext='ZIP', version='1.1')
-def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+
+def _import_common(
+    src_file,
+    temp_dir,
+    instance_data,
+    load_data_callback=None,
+    format_name="yolo",
+    import_kwargs=None,
+    **kwargs
```

Comment:
This parameter seems to be unused.

Reply:
A lot of import functions for some reason have this unused kwargs parameter. I dont know the reason, and that's why I don't wnant to remove it

Reply:
done

Reply:
It's for a backward compatibility option, used in some formats
https://github.com/cvat-ai/cvat/blob/52ab96e3f5069c10ccab9cdaee2fa2457a02e9ee/cvat/apps/dataset_manager/formats/pascal_voc.py#L59